src/server/routes/sitemap.xml.ts

```diff
@@ -0,0 +1,13 @@
+import { SitemapStream, streamToPromise } from "sitemap";
+import routes from "../../../config/routes";
```

Comment:
If we use SSG strategy, we can simply glob the `dist/**/*.html` files as routes. But for SSR this is convenient.